packages/bff/vitest.config.ts

```diff
@@ -0,0 +1,9 @@
+import { defineConfig } from "vitest/config";
+
+export default defineConfig({
+  test: {
+    testTimeout: 60000,
+    hookTimeout: 60000,
+    setupFiles: ["dotenv-flow/config"],
+  },
+});
```

Comment:
This looks like the old `vitest.config.ts` that is still used in some services that were not yet updated (e.g., Catalog, probably Tenant), the new one to be used in new services should be: 
```suggestion
import { defineConfig } from "vitest/config";

export default defineConfig({
  test: {
    globalSetup: ["./test/vitestGlobalSetup.ts"],
    testTimeout: 60000,
    hookTimeout: 60000,
    fileParallelism: false,
    pool: "forks"
  },
});
```